libs/protocol-ui/src/components/molecules/Input/Input.tsx

```diff
@@ -64,6 +64,7 @@ const Input: React.FC<InputProps> = ({
             variant={variant}
             isDisabled={isDisabled}
             {...register(name)}
+            data-testid="chakraInput-test"
```

Comment:
In general lets add a dataTestId prop so we can point to the specific element rather than getting all elements with this id